### PR TITLE
Feature/sg 568 single epoch sanity test for all recipes

### PR DESCRIPTION
**Pull request by shaydeci**
- Updated recipe configs of crashing recipes.
- New jobs and workflows for each regression test by task.

### CODE REVIEW

File: src/super_gradients/recipes/coco2017_ssd_lite_mobilenet_v2.yaml [hunk 1]
```diff
@@ -35,10 +35,10 @@ architecture: ssd_lite_mobilenet_v2
 
 data_loader_num_workers: 8
 
-experiment_suffix: res${dataset_params.train_image_size}
+experiment_suffix: res320
```
Comment (ofrimasad) [inline src/super_gradients/recipes/coco2017_ssd_lite_mobilenet_v2.yaml:38]: are these changes intentional?

File: src/super_gradients/recipes/coco_segmentation_shelfnet_lw.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ val_dataloader: coco_segmentation_val
 
 checkpoint_params:
   strict_load: True
-  load_backbone: True
+  load_backbone: False
```
Comment (ofrimasad) [inline src/super_gradients/recipes/coco_segmentation_shelfnet_lw.yaml:29]: is this intentional?

**Review by ofrimasad — COMMENTED** [review]
just some minor questions.
IF we can shorten the path to the test script somehow, that would be great

File: src/super_gradients/recipes/coco2017_ssd_lite_mobilenet_v2.yaml (hunk 1 above)
Comment (shaydeci) [reply src/super_gradients/recipes/coco2017_ssd_lite_mobilenet_v2.yaml:38]: Yes, this interpolation is not up to date (we removed all of these dataset_params.train_image_size when we got rid of datasetinterface)

**Review by ofrimasad — APPROVED** [review]
LGTM
